packages/docs/src/components/header/components/HeaderSelector.vue

```diff
@@ -44,12 +44,12 @@ export default defineComponent({
   },
 
   setup (props, { emit }) {
-    const { getColors } = useColors()
+    const { colors } = useColors()
 
     const toggleSidebar = () => emit('toggleSidebar', !props.minimized)
 
     return {
-      colors: computed(getColors),
+      colors,
```

Comment:
@aluarius, thank you very much for your help. I take a new way to solve issue 2266. There are some places to change in similar way, So I have pick one place as an example for discussion. I have changed `colors: computed(getColors)` to `colors`, is this the intended solution?

Reply:
No, as was said in the initial issue `computed with get and set instead of two methods`. Now we have getColors and setColors, and we need to unite them. In the last example you have just renamed one of the methods.

Your first implementation was on the right way, unlike this one.